src/app/containers/ContactsContainer.js

```diff
@@ -93,6 +100,8 @@ const ContactsContainer = ({ location, history }) => {
         }, Object.create(null));
     }, [contactEmails]);
 
+    const contactGroupsMap = useMemo(() => toMap(contactGroups && contactGroups.filter(Boolean)), [contactGroups]);
```

Comment:
I can see you just moved this line, but it made me wonder why `contactGroups.filter(Boolean)` is needed? My understanding is that it's a direct model from the API, so it should either be undefined or an array, and if it's an array it shouldn't contain undefined entries.

Reply:
Yep, it seems it's not needed